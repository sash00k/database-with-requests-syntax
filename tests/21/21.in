get records where department is "ММФ" as table1
get records from table1 where group is 22129