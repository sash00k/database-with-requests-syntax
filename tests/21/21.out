Student Иванов С. В. from group 22129, ММФ
Student Иванова В. В. from group 22129, ММФ